sample-code/src/langchain-orchestration.ts

```diff
@@ -187,6 +190,49 @@ export async function invokeLangGraphChain(): Promise<string> {
   return `${firstResponse}\n\n${secondResponse}`;
 }
 
+/**
+ * Stream responses from the OrchestrationClient using LangChain.
+ * @param controller - The abort controller to cancel the request if needed.
+ * @returns An async iterable of AIMessageChunk objects.
+ */
+export async function streamOrchestrationLangChain(
+  controller = new AbortController()
+): Promise<AsyncIterable<OrchestrationMessageChunk>> {
+  const orchestrationConfig: LangchainOrchestrationModuleConfig = {
+    // define the language model to be used
+    llm: {
+      model_name: 'gpt-4o'
+    },
+    // define the template
+    templating: {
+      template: [
+        {
+          role: 'user',
+          content: 'Write a 100 word explanation about {{?topic}}'
+        }
+      ]
+    }
+  };
+
+  const client = new OrchestrationClient(orchestrationConfig);
+
+  // Return the stream
+  return client.stream(
+    [
+      {
+        role: 'user',
+        content: 'I need information about a topic.'
+      }
+    ],
```

Comment:
[pp] I would just leave messages (aka. message history) empty as after the make template optional PR, we will conceptually avoid having template and messages defined at the same time. Either template for one time chat message or messages (same as in LangChain design) for conversational chat completion.

Reply:
Good catch, I would assume in the future we then change this sample to exclude templating and go with just messages instead. (Maybe add as a todo for some follow-up)

Reply:
For now left message history empty, left a todo to update the test after BLI.